### DISCUSSION TITLE
Proxying AdGuard Home via Nginx - 403 Forbidden

### DISCUSSION THREAD

**Discussion by PanSzelescik** [AdguardTeam/AdGuardHome#5277] [Q&A]
Hello, I have AdGuard Home installed in Docker on my VPS. it works very well using numeric IP.
However, I would like to redirect traffic to the admin panel via Nginx also installed in Docker.
I found [this setting](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#nginx) for Nginx, but I get redirect to /aghome/login.html and then 403 Forbidden: 
![image](https://user-images.githubusercontent.com/26679785/208267665-12b37187-b7fe-4be3-bd6e-fe7a7616eb2c.png)
![image](https://user-images.githubusercontent.com/26679785/208267724-2b3d5c51-7021-4ca9-a3f8-c758d3543c1f.png)
Have any ideas?

**Comment by ammnt** [comment]
> Hello, I have AdGuard Home installed in Docker on my VPS. it works very well using numeric IP.
> However, I would like to redirect traffic to the admin panel via Nginx also installed in Docker.
> I found [this setting](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#nginx) for Nginx, but I get redirect to /aghome/login.html and then 403 Forbidden:
> ![image](https://user-images.githubusercontent.com/26679785/208267665-12b37187-b7fe-4be3-bd6e-fe7a7616eb2c.png)
> ![image](https://user-images.githubusercontent.com/26679785/208267724-2b3d5c51-7021-4ca9-a3f8-c758d3543c1f.png)
> Have any ideas?

Try to remove `/aghome/` subfolder in `proxy_redirect` and `proxy_cookie_path` directives. I think there is no `/aghome/` directory. What I have now in my Docker Compose config file:

location / {
    proxy_pass http://172.16.0.2:80;
    proxy_set_header Host $host;
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_set_header X-Forwarded-Host $host;
    proxy_set_header X-Forwarded-Server $host;
    proxy_set_header X-Forwarded-Port $server_port;
    proxy_redirect off;
    proxy_cookie_path / /;
}
